data/map planets.txt

```diff
@@ -404,6 +404,7 @@ planet Bluerun
 	attributes hai station uninhabited
 	landscape land/station33
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	Bluerun seems to have been mainly built for civilian habitation, but extensive mineral processing facilities indicate that ore refining was once a sizable industry here. No doubt the designers of this station sought to take advantage of the large number of metallic asteroids in this system and process ores stationside rather than rely solely on Stonebreak's facilities. Doing so presumably reduced the resource burden caused by the massive influx of migrants to this station.`
```

Comment:
```suggestion
	description `	Bluerun seems to have been mainly built for civilian habitation, but extensive mineral processing facilities indicate that ore refining was once a sizable industry here. No doubt the designers of this station sought to take advantage of the large number of metallic asteroids in this system and process ores stationside.`
```
I personally think the explanation in the last two sentences is a little excessive. It's so much historical context that it feels almost immersion-breaking given that we're supposed to be viewing this from `first last`'s perspective, not an omniscient one. I don't mean to knock any of the thought that went into this - I think this is interesting and makes sense, but as-is it just feels like a little too much to feed the player at once for snooping around on a derelict station. If y'all have other thoughts on how to trim this down or if you disagree, I'm open to suggestions and thoughts!